QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -100,10 +115,29 @@ public void event() {
 	private static EventOutput calibrate = new EventOutput() {
 		public void event() {
 			float yaw = HeadingSensor.yaw.get();
+			desiredAngle.set(yaw);
+			if (Igneous.getIsDisabled().get()){
```

Comment:
You should stick `Igneous.getIsDisable()` into a variable - you don't want it to be creating a new channel for you every single time.